test/rmq/jupyter-test.py

```diff
@@ -0,0 +1,23 @@
+# %%
+
+import kiwipy
```

Comment:
Is there a particular reason why you have this as a free-standing script and not as a test function?